CodeEdit/Utils/TabView/AnyVariadicView+Extensions.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  AnyVariadicView+Extensions.swift
+//  CodeEdit
+//
+//  Created by Wouter on 27/12/23.
+//
+
+import Engine
+import UniformTypeIdentifiers
+
+extension AnyVariadicView.Subview {
+    var onMove: ((IndexSet, Int) -> Void)? {
+        self["s7SwiftUI14OnMoveTraitKeyV", as: ((IndexSet, Int) -> Void).self]
```

Comment:
These are mad sketchy, can we add a test for this so we can be notified quickly if they change in a future SwiftUI release?